spec/examples.txt

```diff
@@ -0,0 +1,13 @@
+example_id                                                    | status | run_time        |
```

Comment:
Not sure what this file is for and if it should be in VCS ?

Reply:
Nor I, it only appeared after I renamed the `features` folder. Added to `.gitignore` and untracked.